wallet/src/libwallet/internal/restore.rs

```diff
@@ -215,6 +217,12 @@ where
 				is_coinbase: output.is_coinbase,
 				tx_log_entry: None,
 			});
+
+			max_child_index = if max_child_index >= output.n_child.unwrap() {
+				max_child_index
+			}else{
```

Comment:
Nitpick: we use `} else {`. Also do you have `rustfmt` configured to run on commit? It would catch all these little details.

Reply:
thanks. always forgot to link that githook in a new clone, will find auto way to have rustfmt for us